cmd/agent/subcommands/diagnose/command.go

```diff
@@ -267,19 +267,8 @@ This command print the security-agent metadata payload. This payload is used by
 		},
 	}
 
-	showPayloadCommand.AddCommand(payloadV5Cmd)
-	showPayloadCommand.AddCommand(payloadGohaiCmd)
-	showPayloadCommand.AddCommand(payloadInventoriesAgentCmd)
-	showPayloadCommand.AddCommand(payloadInventoriesHostCmd)
-	showPayloadCommand.AddCommand(payloadInventoriesOtelCmd)
-	showPayloadCommand.AddCommand(payloadInventoriesChecksCmd)
-	showPayloadCommand.AddCommand(payloadInventoriesPkgSigningCmd)
-	showPayloadCommand.AddCommand(payloadSystemProbeCmd)
-	showPayloadCommand.AddCommand(payloadSecurityAgentCmd)
-	diagnoseCommand.AddCommand(showPayloadCommand)
-
 	showAgentTelemetryCommand := &cobra.Command{
-		Use:   "show-telemetry",
+		Use:   "agent-telemetry",
```

Comment:
Note: This is a breaking change. I guess this should be OK as the command is very new.

Reply:
is not a breaking change as the command is not release yet. This PR uses this PR as the starting point https://github.com/DataDog/datadog-agent/pull/29770